frontend/src/views/WheresMyGene/components/HeatMap/components/Chart/index.tsx

```diff
@@ -304,9 +302,14 @@ export default memo(function Chart({
               y: cursorOffset[1],
             }),
           },
-          modifiers: {
-            offset: { offset: "0,20" },
-          },
+          modifiers: [
+            {
+              name: "offset",
+              options: {
+                offset: [0, 5],
```

Comment:
Is this changing the placement of the tooltip relative to the cursor?

Reply:
Great eye! I had to adjust the values a little because the original values were way off in Popper2